### ISSUE TITLE
Add acceleration test components and associated reference components

### ISSUE THREAD

**Issue by ronald-cron-arm** [Mbed-TLS/TF-PSA-Crypto#62]
Add in TF-PSA-Crypto the equivalent of the test_psa_crypto_config_accel.* and test_psa_crypto_config_reference.* components.

`test_psa_crypto_config_accel_ecdsa` not in scope of this issue (part of #61).

Depends on #61.
